themes/default/content/docs/using-pulumi/adopting-pulumi/converters.md

```diff
@@ -13,38 +13,39 @@ aliases:
-        CloudFormation to Pulumi:
-        </a>
-        This conversion tool will do the magic of translating your CloudFormation templates into TypeScript/JavaScript, Python, Go and C# using Pulumi.
-    </li>
-    <li class="pt-4">
-        <a class="" href="/blog/introducing-crd2pulumi">
-        Kubernetes CustomResources to Pulumi:
-        </a>
-        CustomResources in Kubernetes allow users to extend the API with their types. These types are defined using CustomResourceDefinitions (CRDs), which include an OpenAPI schema.
-        Our new crd2pulumi tool takes the pain out of managing CustomResources by generating types in the Pulumi-supported language of your choice!
-    </li>
-    <li class="pt-4">
-        <a class="" href="/kube2pulumi">
-        Kubernetes YAML to Pulumi:
-        </a>
-        This conversion tool will do the magic of translating your YAML into modern code using Pulumi.
-    </li>
-    <li class="pt-4">
-        <a class="" href="/tf2pulumi">
-        Terraform to Pulumi:
-        </a>
-        This conversion tool will do the magic of translating your HCL into modern code using Pulumi.
-    </li>
-</ul>
+`pulumi convert` is a subcommand within the Pulumi Command Line Interface (CLI) you can use to convert between different source IaC languages or formats into all of Pulumi’s [supported languages](/docs/languages-sdks/), such as TypeScript, JavaScript, Go, Python, C#, or Java.
+
+For the detailed usage of this command and options, refer to the [pulumi convert CLI documentation.](https://www.pulumi.com/docs/cli/commands/pulumi_convert/)
+
+### Supported source languages
+
+* [YAML](/docs/languages-sdks/yaml/)
+* [Terraform](/docs/using-pulumi/adopting-pulumi/migrating-to-pulumi/from-terraform/)
+* [Kubernetes](/docs/using-pulumi/adopting-pulumi/migrating-to-pulumi/from-kubernetes/)
+* [ARM](/docs/using-pulumi/adopting-pulumi/migrating-to-pulumi/from-azure)
+* [Bicep](/docs/using-pulumi/adopting-pulumi/migrating-to-pulumi/from-azure)
+
+### Supported destination languages
+
+* [TypeScript/Javascript](/docs/languages-sdks/javascript/)
+* [Go](/docs/languages-sdks/go/)
+* [Python](/docs/languages-sdks/python/)
+* [C#](/docs/languages-sdks/dotnet/)
+* [Java](/docs/languages-sdks/java/)
+
+### Use cases for using pulumi convert
+
+The `pulumi convert` subcommand is designed to address a variety of migration and conversion scenarios where infrastructure as code (IaC) practices evolve alongside your project needs. Here are some common use cases:
+
+* **Migrating from other IaC tools:** If you're looking to transition your infrastructure definitions from tools like Terraform, Bicep, or ARM templates into Pulumi.
+* **Converting from Pulumi YAML:** If you started your project with Pulumi YAML and find your requirements have evolved, you can convert and take advantage of the benefits offered by popular programming languages, such as complex logic capabilities and the flexibility to create modular and reusable components.
```

Comment:
Re: the "modular and resuable components" part, is this a reference to [Pulumi Components](https://www.pulumi.com/docs/concepts/resources/components/)? If so, should we link it here?